plover/gui_qt/add_translation_widget.py

```diff
@@ -6,14 +6,16 @@
 from PyQt5.QtCore import QEvent, QTimer
 from PyQt5.QtWidgets import QApplication, QWidget
 
+from plover import _
+
```

Comment:
Please remove the extra empty line.

Reply:
Oops, missed one! I got it.

(The newline formatting isn't terribly consistent in most of the code, so I'm surprised anyone cared...but this one would have bothered me too. Good catch!)